### PR TITLE
SorrTask1038_Test_get_range_filter

### PR DESCRIPTION
**Pull request by smitpatel49**
#1038

### CODE REVIEW

**Comment by smitpatel49** [comment]
@samarth9008 , awaiting review on this one.

File: oms/child_order_quantity_computer/test/test_scheduling_child_order_quantity_computers.py [hunk 1]
```diff
@@ -81,6 +81,20 @@ def test_get_wave_quantities4(self) -> None:
             {1: 1.0, 2: 0.3, 3: 1.0},
         )
 
+    def test_get_range_filter(self) -> None:
+        """
+        Test that the range of expected number of child orders.
```
Comment (gpsaggese) [inline oms/child_order_quantity_computer/test/test_scheduling_child_order_quantity_computers.py:86]: Improve English pls

File: oms/child_order_quantity_computer/test/test_scheduling_child_order_quantity_computers.py [hunk 2]
```diff
@@ -81,6 +81,20 @@ def test_get_wave_quantities4(self) -> None:
...
+        """
+        # Define the parameters.
```
Comment (gpsaggese) [inline oms/child_order_quantity_computer/test/test_scheduling_child_order_quantity_computers.py:88]: Set inputs.

Test

Check

Comment (smitpatel49) [reply oms/child_order_quantity_computer/test/test_scheduling_child_order_quantity_computers.py:88]: I have changed the doc string for the function along with the comment for inputs. But I am not sure about what you mean by Test and Check.

Comment (DanilYachmenev) [reply oms/child_order_quantity_computer/test/test_scheduling_child_order_quantity_computers.py]: These are comment sections of a test
```
# Set inputs.
...
# Test.
...
# Check outputs.
...
```

File: oms/child_order_quantity_computer/test/test_scheduling_child_order_quantity_computers.py [hunk 3]
```diff
@@ -81,6 +81,20 @@ def test_get_wave_quantities4(self) -> None:
             {1: 1.0, 2: 0.3, 3: 1.0},
         )
 
+    def test_get_range_filter(self) -> None:
+        """
+        Test that the range of child orders are returned as expected.
```
Comment (DanilYachmenev) [inline oms/child_order_quantity_computer/test/test_scheduling_child_order_quantity_computers.py:86]: -> `Check that the correct range of expected child orders is returned.`

**Review by DanilYachmenev — COMMENTED** [review]
lgtm

**Review by gpsaggese — COMMENTED** [review]
LGTM

**Review by DanilYachmenev — APPROVED** [review]

**Pull request merged**